comms/lighthouse/src/peerjs-server/services/messagesExpire/index.ts

```diff
@@ -37,8 +37,12 @@ export class MessagesExpire implements IMessagesExpire {
     }
 
     // Clean up outstanding messages
-    this.timeoutId = setTimeout(() => {
-      this.pruneOutstanding()
+    this.timeoutId = setTimeout(async () => {
```

Comment:
Same as first comment, setTimeout should have a function returning void, it won't handle the catch from the Promise

I'm thinking that a high order function to wrap "async callbacks" and returns void functions may be a good idea.

Reply:
This one is try-catched, so it shouldn't be a problem. But maybe I should propagate the error somehow? I think it is not necessary, though.

Reply:
Ok sounds good, I'd do something to start getting used to it, this is something that would break the program in a future. But not this specific case, due to the "catch"

```
Received has value: undefined (node:32535) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:32535) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```